src/main/java/de/tum/in/www1/artemis/domain/FileUploadExercise.java

```diff
@@ -1,6 +1,9 @@
 package de.tum.in.www1.artemis.domain;
 
-import javax.persistence.*;
+import javax.persistence.Column;
+import javax.persistence.DiscriminatorValue;
+import javax.persistence.Entity;
+import javax.persistence.Lob;
```

Comment:
The changes in this file don't seem to be related to the PR(?)

Reply:
Seems to be just a reformating change. Not really a problem right?

Reply:
It is a problem, please avoid it! It's not good if PRs randomly change the imports due to different IntelliJ settings.
@MaisiKoleni recently added some configs that should be synced directly with IntelliJ.
Please make sure to have the correct project settings in IntelliJ to avoid it.